utils/index.d.ts

```diff
@@ -1599,3 +1599,78 @@ export declare class DeleteConfirmationStep extends AzureWizardPromptStep<IActio
  * @returns a sorted, unique string of values separated by `;`
  */
 export function createContextValue(values: string[]): string;
+
+/**
+ * Gets a normalized type for an Azure resource, accounting for the fact that some
+ * Azure resources share values for type and/or kind
+ * @param resource The resource to check the {@link AzExtResourceType} for
+ * @returns The normalized Azure resource type
+ */
+export declare function getAzExtResourceType(resource: { type: string; kind?: string }): AzExtResourceType | undefined;
+
+/**
+ * Normalized type for Azure resources that uniquely identifies resource type for the purposes
+ * of the Azure extensions
+ */
+export declare enum AzExtResourceType {
```

Comment:
While these remain duplicated, we should make sure there are comments pointing one to the other.